Packs/MalwareBazaar/Integrations/MalwareBazaarFeed/MalwareBazaarFeed.yml

```
@@ -0,0 +1,128 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: MalwareBazaar Feed
+  version: -1
+configuration:
+- display: Malware Bazaar API URL
```

no need to state malware bazzar again.
```suggestion
- display: Server URL
```